File: apps/institution-api/Models/VwCurrentInstitutionLocationPhone.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class VwCurrentInstitutionLocationPhone
    {
        public int? InstitutionLocationPhoneId { get; set; }
        public int InstitutionId { get; set; }
        public int InstitutionLocationId { get; set; }
        public string InstitutionName { get; set; }
        public string InstitutionCode { get; set; }
        public string InstitutionStatus { get; set; }
        public string LocationName { get; set; }
        public int? LocationNumber { get; set; }
        public string LocationStatus { get; set; }
        public string PhoneUsage { get; set; }
        public string PhoneType { get; set; }
        public int? ListSequence { get; set; }
        public string CountryCallingCode { get; set; }
        public string PhoneNumber { get; set; }
        public string Extension { get; set; }
        public Guid? PhoneGuid { get; set; }
    }
}

File: libs/common/models/vw-current-institution-location-phone.ts



export class VwCurrentInstitutionLocationPhone {
    public institutionLocationPhoneId: number;
    public institutionId: number = 0;
    public institutionLocationId: number = 0;
    public institutionName: string = '';
    public institutionCode: string = '';
    public institutionStatus: string = '';
    public locationName: string = '';
    public locationNumber: number;
    public locationStatus: string = '';
    public phoneUsage: string = '';
    public phoneType: string = '';
    public listSequence: number;
    public countryCallingCode: string = '';
    public phoneNumber: string = '';
    public extension: string = '';
    public phoneGuid: string;

    constructor(init? : Partial<VwCurrentInstitutionLocationPhone>) {
        
        Object.assign(this, init);
    }
}

